docs/content/installation/integrations/app-protect-waf/compile-waf-policies.md

```diff
@@ -0,0 +1,359 @@
+
+If the wrong version is noted in the JSON payload, you will receive an error similar to below:
+
+```text
+{"code":13018,"message":"Error compiling the security policy set: One or more of the specified compiler versions does not exist. Check the compiler versions, then try again."}
+```
+
+Create the file `security-policy-bundles.json`:
+
+```json
+{
+  "bundles": [
+    {
+      "appProtectWAFVersion": "4.815.0",
+      "policyName": "Nginxbundletest",
+      "policyUID": "",
+      "attackSignatureVersionDateTime": "latest",
+      "threatCampaignVersionDateTime": "latest"
+    }
+  ]
+}
+```
+
+The *policyUID* value is left blank, as it is generated as part of the creating the bundle.
+
+Send a POST request to create the bundle through the API:
+
+```shell
+curl -X POST https://{{NMS_FQDN}}/api/platform/v1/security/policies/bundles \
+    -H "Authorization: Bearer <access token>" \
+    -d @security-policy-bundles.json
+```
+
+You should receive a response similar to the following:
+
+```json
+{
+    "items": [
+        {
+            "compilationStatus": {
+                "message": "",
+                "status": "compiling"
+            },
+            "content": "",
+            "metadata": {
+                "appProtectWAFVersion": "4.815.0",
+                "attackSignatureVersionDateTime": "2024.02.21",
+                "created": "2024-06-12T13:28:20.023775785-07:00",
+                "modified": "2024-06-12T13:28:20.023775785-07:00",
+                "policyName": "Nginxbundletest",
+                "policyUID": "6af9f261-658b-4be1-b07a-cebd83e917a1",
+                "threatCampaignVersionDateTime": "2024.02.25",
+                "uid": "cbdf9577-6d81-43d6-8ce1-2e3d4714e8b5"
+            }
+        }
+    ]
+}
+```
+
+You can use the API to list the security bundles, verifying the new addition:
+
+```shell
+curl --location 'https://127.0.0.1/api/platform/v1/security/policies/bundles' \
+-H "Authorization: Bearer <access_token>"
+```
+```json
+{
+    "items": [
+        {
+            "compilationStatus": {
+                "message": "",
+                "status": "compiled"
+            },
+            "content": "",
+            "metadata": {
+                "appProtectWAFVersion": "4.815.0",
+                "attackSignatureVersionDateTime": "2024.02.21",
+                "created": "2024-06-13T09:09:10.809-07:00",
+                "modified": "2024-06-13T09:09:20-07:00",
+                "policyName": "Nginxbundletest",
+                "policyUID": "ec8681eb-1e25-4b71-93bd-b91f67c5ac99",
+                "threatCampaignVersionDateTime": "2024.02.25",
+                "uid": "de08b324-99d8-4155-b2eb-fe687b21034e"
+            }
+        }
+    ]
+}
+```
+
+{{< important >}}
+
+Take note of the *uid* field. 
+
+It is one of two unique IDs we will use to download the bundle: it will be referenced as *bundle-UID*.
+
+{{< /important >}}
```

Comment:
```suggestion
{{< important >}}

Take note of the *uid* field: "uid": "de08b324-99d8-4155-b2eb-fe687b21034e"

It is one of two unique IDs we will use to download the bundle: it will be referenced as *bundle-UID*.

{{< /important >}}
```